modules/tinymce/src/core/main/ts/init/InitContentBody.ts

```diff
@@ -288,15 +288,21 @@ const loadContentCss = (editor: Editor, css: string[]) => {
   const styleSheetLoader = getStyleSheetLoader(editor);
   const fontCss = Settings.getFontCss(editor);
 
-  const loaded = () => {
-    editor.on('remove', () => {
-      styleSheetLoader.unloadAll(css);
+  const removeCss = () => {
+    styleSheetLoader.unloadAll(css);
 
-      if (!editor.inline) {
-        editor.ui.styleSheetLoader.unloadAll(fontCss);
-      }
-    });
-    initEditor(editor);
+    if (!editor.inline) {
+      editor.ui.styleSheetLoader.unloadAll(fontCss);
+    }
+  };
+
+  const loaded = () => {
+    if (editor.destroyed) {
```

Comment:
This should probably be `editor.removed` instead, as that's set when it's flagged for removal, whereas as `editor.destroyed` is when it's completed killed.

Reply:
Done